internal/tiltfile/tiltfile_state.go

```diff
@@ -427,7 +427,7 @@ func (s *tiltfileState) potentiallyK8sUnsafeBuiltin(f starkit.Function) starkit.
 			kubeContext := k8sContextState.KubeContext()
 			return nil, fmt.Errorf(`Refusing to run '%s' because %s might be a production kube context.
 If you're sure you want to continue add:
-allow_k8s_contexts('%s')
+	allow_k8s_contexts('%s')
```

Comment:
was this intentional?

Reply:
yeah, to set apart the command to run with a tab!

Reply:
...except whoops this is the wrong PR 😅